src/browser/webview/mattermost.js

```diff
@@ -47,6 +47,12 @@ window.addEventListener('load', () => {
   });
 });
 
+if (process.platform !== 'darwin') {
+  window.addEventListener('mousedown', () => {
+    ipcRenderer.sendToHost('closeMenu');
+  });
+}
```

Comment:
maybe I'm missing something, but are we sending a `closemenu` event every time the user clicks on linux and windows?
what's the reason for that behaviour?

Reply:
It's so the `MattermostView` can send an event back to the index to close the top bar menu on Windows.